PostBoard.Api/Controllers/BirthdayController.cs

```diff
@@ -70,29 +80,32 @@ public IActionResult Update([FromRoute] int id, [FromBody] Birthday input)
             return BadRequest("Validation error.");
         }
 
-        var birthday = Birthdays.FirstOrDefault(b => b.Id == id);
+        var birthday = _context.Birthdays.FirstOrDefault(b => b.Id == id);
         if (birthday == null)
         {
             return NotFound($"Birthday with Id={id} not found.");
         }
 
         birthday.UserFullName = input.UserFullName;
         birthday.Date = input.Date;
-        
+        _context.Entry(birthday).State = EntityState.Modified;
```

Comment:
This line is not necessary. In `GetById` request you don't modify entity, so there is no need to set `Modified` state for it's entry.
Remove this line.